.github/workflows/build.yaml

```diff
@@ -0,0 +1,49 @@
+name: ci
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
```

Comment:
Why is this required?

Reply:
Ah. Is it a requirement to build for arm etc?

Reply:
Yes, in combination with buildx you can build images for arm as well.